.gitignore

```diff
@@ -110,6 +110,7 @@ drape_frontend/shader_def.hpp
 drape_frontend/shader_def.cpp
 drape_frontend/drape_frontend_tests/shader_def_for_tests.hpp
 drape_frontend/drape_frontend_tests/shader_def_for_tests.cpp
+shaders_compiler/
```

Comment:
tools/shaders_compiler is a valid directory. Why do you need this line in .gitignore?

Reply:
Shoot, I missed that! I most likely got that directory from not building on another directory, like the rest.